pallets/parachain-staking/src/lib.rs

```diff
@@ -1268,6 +1269,10 @@ pub mod pallet {
 			more: BalanceOf<T>,
 		) -> DispatchResultWithPostInfo {
 			let delegator = ensure_signed(origin)?;
+			ensure!(
+				!Self::delegation_request_revoke_exists(&candidate, &delegator),
```

Comment:
The benchmark for this function needs to be updated no? This implies an additional DB read now

Reply:
Yes, the benchmark is running as of now. Also we need a way to fix the two accounts that didn't get their balance unreserved on alphabet. What could be a good way for it?